dotCMS/src/main/java/com/dotmarketing/portlets/workflows/model/WorkflowStatus.java

```diff
@@ -31,16 +34,21 @@ public static String toCommaSeparatedString(final Set<WorkflowStatus> workflowSt
      * @param value Object
      * @return Set of {@link WorkflowStatus}
      */
-    public static  Set<WorkflowStatus> toSet(Object value) {
+    public static  Set<WorkflowStatus> toSet(final Object value) {
 
         Set<WorkflowStatus> workflowStatusSet = Collections.emptySet();
 
-        if (null != value) {
-
-            workflowStatusSet = Stream
-                    .<String>of(value.toString().split(DELIMITER))
-                    .map(WorkflowStatus::valueOf)
-                    .collect(Collectors.toSet());
+        if (null != value && UtilMethods.isSet(value.toString())) {
+
+            try {
+                workflowStatusSet = Stream
+                        .<String>of(value.toString().split(DELIMITER))
+                        .map(WorkflowStatus::valueOf)
+                        .collect(Collectors.toSet());
+            } catch (Exception e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)

Reply:
no specific one, so